pkg/installation/url.go

```diff
@@ -35,7 +35,8 @@ func GetUrlType(u string) int {
 }
 
 func isK8sApiUrl(u string) bool {
-	return k8sApiURLRegexp.MatchString(u)
+	u = strings.SplitN(u, ":", 2)[0]
+	return k8sApiURLRegexp.MatchString(u) || (strings.HasPrefix(u, "api") && strings.HasSuffix(u, "gigantic.io"))
```

Comment:
Probably doesn't matter that much, but this would match invalid strings such as `apigigantic.io` or `api.gigantic.io`.